submission.yaml

```diff
@@ -1 +1,1143 @@
+    nodes:
+    -   id: MESH:C005430
+        label: Drug
+        name: Phenindamine
+    -   id: UniProt:P35367
+        label: Protein
+        name: Histamine H1 receptor
+    -   id: MESH:D006632
+        label: ChemicalSubstance
+        name: Histamine
+    -   id: GO:0002524
+        label: BiologicalProcess
+        name: Hypersensitivity
+    -   id: MESH:D012223
+        label: Disease
+        name: Vasomotor rhinitis
+    reference:
+    - https://go.drugbank.com/drugs/DB01619
+    - https://pubchem.ncbi.nlm.nih.gov/compound/Phenindamine#section=MeSH-Pharmacological-Classification
+    - https://en.wikipedia.org/wiki/Antihistamine
+-   directed: true
+    graph:
+        drug: Phenindamine
+        drugbank: DB:DB01619
+        drug_mesh: MESH:C005430
+        disease: Allergic conjunctivitis
+        disease_mesh: MESH:D003233
+    links:
+    -   key: decreases activity of
+        source: MESH:C005430
+        target: UniProt:P35367
+    -   key: positively correlated with
+        source: UniProt:P35367
+        target: MESH:D006632
+    -   key: causes
+        source: MESH:D006632
+        target: GO:0002524
+    -   key: correlated with
+        source: GO:0002524
+        target: MESH:D003233
+    multigraph: true
+    nodes:
+    -   id: MESH:C005430
+        label: Drug
+        name: Phenindamine
+    -   id: UniProt:P35367
+        label: Protein
+        name: Histamine H1 receptor
+    -   id: MESH:D006632
+        label: ChemicalSubstance
+        name: Histamine
+    -   id: GO:0002524
+        label: BiologicalProcess
+        name: Hypersensitivity
+    -   id: MESH:D003233
+        label: Disease
+        name: Allergic conjunctivitis
+    reference:
+    - https://go.drugbank.com/drugs/DB01619
+    - https://pubchem.ncbi.nlm.nih.gov/compound/Phenindamine#section=MeSH-Pharmacological-Classification
+    - https://en.wikipedia.org/wiki/Antihistamine
+-   directed: true
+    graph:
+        drug: Oxitropium
+        drugbank: DB:DB12086
+        drug_mesh: MESH:C017590
+        disease: Chronic obstructive lung disease
+        disease_mesh: MESH:D029424
+    links:
+    -   key: decreases activity of
+        source: MESH:C017590
+        target: UniProt:P20309
+    -   key: causes
+        source: UniProt:P20309
+        target: GO:0006939
+    -   key: contributes to
+        source: GO:0006939
+        target: MESH:D029424
+    multigraph: true
+    nodes:
+    -   id: MESH:C017590
+        label: Drug
+        name: Oxitropium
+    -   id: UniProt:P20309
+        label: Protein
+        name: Muscarinic acetylcholine receptor M3
+    -   id: GO:0006939
+        label: BiologicalProcess
+        name: Smooth muscle contraction
+    -   id: MESH:D029424
+        label: Disease
+        name: Chronic obstructive lung disease
+    reference:
+    - https://go.drugbank.com/drugs/DB12086
+    - https://drugs.ncats.io/drug/8G15T83E6I
+    comment: Oxitropium is used as Oxitropium bromide for treatment
 
```

Comment:
@umasrisankarlal could you remove that space in line 282 please?

Reply:
@TanskaAnnna , corrected the changes, but the PR shows failed.
Can you please check and suggest?

Reply:
@umasrisankarlal it failed because you have changes in more than one file. Only changes to the submission.yaml are allowed.

Reply:
Hello @TanskaAnnna , I deleted my fork and created a new one. New one is updated with recent changes